src/utils/paramParseHelpers.ts

```diff
@@ -0,0 +1,17 @@
+import { AssessmentCategory } from '../components/assessment/assessmentShape'
+
+/*
+ * Converts an AssessmentCategory into a string for use in URLs.
+ * E.g Mission -> missions
+ */
+export const assessmentCategoryLink = (cat: AssessmentCategory): string =>
+  cat.toLowerCase().concat('s')
+
+/* Converts an optinal string parameter into an integer or null value. */
```

Comment:
Should standardise function docs with jsdocs style comments:

http://usejsdoc.org/about-getting-started.html